waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3884 @@
+"Gatterlkoepfe",Gattepfe,AT,4723.683N,01100.350E,2475.0m,7,,,,"Gipfel"
+"Gaugen",Gaugen,AT,4647.183N,01312.700E,2195.0m,7,,,,"Gipfel"
+"Gehren-Spitze",Gehretze,AT,4730.100N,01039.383E,2164.0m,7,,,,"Gipfel"
+"Geiberg",Geiberg,AT,4725.599N,01217.333E,1767.0m,7,,,,"Gipfel"
+"Geierkoepfe",Geierpfe,AT,4731.183N,01052.583E,2162.9m,7,,,,"Gipfel"
+"Geierkogel",Geiergel,AT,4724.050N,01429.266E,2237.0m,7,,,,"Gipfel"
+"Geisel-Kogel",Geisegel,AT,4700.783N,01304.416E,2974.0m,7,,,,"Gipfel"
+"Geis-Horn",Geis-orn,AT,4728.166N,01028.650E,2249.0m,7,,,,"Gipfel"
+"Geisshorn",Geisshorn,AT,4717.394N,01010.461E,2365.9m,7,,,,"Gipfel"
+"Geissstein",Geisssein,AT,4720.250N,01229.600E,2363.0m,7,,,,"Gipfel"
+"Gemeindealpe",Gemeilpe,AT,4748.699N,01514.950E,1626.0m,7,,,,"Bergstation"
+"Gennerhorn",Genneorn,AT,4740.504N,01318.688E,1734.9m,7,,,,"Gipfel"
+"Gensgitsch",Gensgsch,AT,4712.650N,01343.916E,2279.0m,7,,,,"Gipfel"
+"Gerlitzen",Gerlizen,AT,4641.716N,01354.933E,1909.0m,7,,,,"Sender"
+"Gerlossteinwand",Gerloand,AT,4711.583N,01155.483E,2166.0m,7,,,,"Gipfel"
+"Gern",Gern,AT,4718.449N,01314.316E,1787.0m,7,,,,"Gipfel"
+"Gernhorn",Gernhorn,AT,4734.683N,01246.883E,1908.0m,7,,,,"Gipfel"
+"Gernkogel",Gernkgel,AT,4715.950N,01211.250E,2267.0m,7,,,,"Gipfel"
+"Gerstingerjoch",Gerstoch,AT,4720.783N,01215.816E,2037.0m,7,,,,"Gipfel"
+"Gerzkopf",Gerzkopf,AT,4727.666N,01325.900E,1729.0m,7,,,,"Gipfel"
+"Gigalitz",Gigalitz,AT,4703.500N,01153.500E,3002.0m,7,,,,"Gipfel"
+"Gilfert-Berg",Gilfeerg,AT,4716.016N,01144.516E,2506.0m,7,,,,"Gipfel"
+"Gimpel",Gimpel,AT,4730.166N,01036.816E,2176.0m,7,,,,"Gipfel"
+"Gippel",Gippel,AT,4747.833N,01534.833E,1669.0m,7,,,,"Gipfel"
+"Gjaidtroghoehe",Gjaidoehe,AT,4703.833N,01254.750E,2989.0m,7,,,,"Gipfel"
+"Glanterer Kogel",Glantgel,AT,4724.300N,01209.700E,1472.0m,7,,,,"Gipfel"
+"Glaserer",Glaserer,AT,4707.300N,01313.016E,2434.0m,7,,,,"Gipfel"
+"Glatthorn",Darmueorn,AT,4715.922N,00952.821E,2133.9m,7,,,,"Gipfel"
+"Glockenkar-Kogl",Glockogl,AT,4705.450N,01210.833E,2913.0m,7,,,,"Gipfel"
+"Glocken-Kogel",Glockgel,AT,4706.900N,01232.633E,2829.0m,7,,,,"Gipfel"
+"Glockhaus",Glockaus,AT,4657.400N,01041.100E,3101.0m,7,,,,"Gipfel"
+"Glockturm",Glockurm,AT,4653.683N,01039.850E,3355.0m,7,,,,"Gipfel"
+"Gloedis-Spitze",Gloeditze,AT,4657.649N,01243.633E,3206.0m,7,,,,"Gipfel"
+"Glungezer",Glungzer,AT,4712.333N,01131.816E,2677.0m,7,,,,"Gipfel"
+"Gmeineck",Huehneerg,AT,4653.764N,01325.797E,2592.0m,7,,,,"Gipfel"
+"Goelbner",Goelbner,AT,4649.466N,01230.316E,2943.0m,7,,,,"Gipfel"
+"Goeller",Goeller,AT,4747.500N,01529.516E,1790.0m,7,,,,"Gipfel"
+"Goesseck",Goesseck,AT,4726.966N,01453.816E,2215.0m,7,,,,"Gipfel"
+"Goldegg",Goldegg,AT,4645.527N,01327.569E,2142.0m,7,,,,"Gipfel"
+"Golz",Golz,AT,4640.316N,01322.066E,2004.0m,7,,,,"Gipfel"
+"Golzentipp",Golzeipp,AT,4644.049N,01236.333E,2317.0m,7,,,,"Gipfel"
+"Gornia",Gornia,AT,4631.183N,01449.133E,1187.0m,7,,,,"Gipfel"
+"Gr Iingent",GrIinent,AT,4703.383N,01147.166E,2917.0m,7,,,,"Gipfel"
```

Comment:
When using abbreviations please be consistent. There are a Gr. and Grosser. I prefer to use the full name.